coc/client.py

```diff
@@ -137,6 +137,9 @@ class Client:
         special flags that will only be interpreted by coc.py if you set this
         bool to True.
 
+    raw_attribute: :class:`bool`
+        The option to enable the _raw_data attribute for classes
```

Comment:
I changed it to
```
The option to enable the _raw_data attribute for most objects in the library. This attribute will contain
the original json data as returned by the API. This can be useful if you want to store a response in a database
for later use or are interested in new things that coc.py does not support otherwise yet. But because this
increases the memory footprint and is not needed for most use cases, this defaults to ``False``.
```
What do you think about that?